### PR TITLE
Add python3 as supported. Isortify. Change test runner

### LINKED ISSUE #75: Switch to python 3
### Motivation
Part of https://github.com/yola/production/issues/4573. We are going to start switch `sitewitservice`  to python 3. The client is a dependency that should work with python 3 as well.

### Todo
- make code python3 compatible

### PR DESCRIPTION
**Pull request by kosmos342**
closes #75

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+collect_ignore = []
+if not os.environ.get('INTEGRATION_TESTS'):
+    collect_ignore.extend([
+        'accounts/test_integration.py',
+        'campaigns/test_integration.py',
+        'partners/test_integration.py',
+    ])
+
+    @pytest.fixture(scope="function", autouse=True)
```
Comment (snitch) [inline tests/conftest.py]: Q000 Remove bad quotes.

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/accounts/base.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def create_account(self, **overrides):
             'site_id': self.site_id,
             'url': self.url,
             'user_name': self.user_name,
-            'user_id': self.user_email,
+            'user_id': '{}@yola.com'.format(uuid4().hex),
```
Comment (Toshakins) [inline tests/accounts/base.py]: Why did you change that line? Was it working correctly before?

File: tests/campaigns/test_integration.py [hunk 3]
```diff
@@ -23,8 +27,8 @@ def setUpClass(cls):
             'http://www.test.site.com', email, 'Foo', email, 'USD', 'US'
         )['accountInfo']['token']
 
-        cls.subscribe_method = cls.subscribe_method()
```
Comment (Toshakins) [inline tests/campaigns/test_integration.py]: 😕

File: tests/campaigns/test_integration.py [hunk 4]
```diff
@@ -23,8 +27,8 @@ def setUpClass(cls):
...
-        cls.unsubscribe_method = cls.unsubscribe_method()
+        cls.subscribe_method = cls.get_subscribe_method()
```
Comment (Toshakins) [inline tests/campaigns/test_integration.py]: 👍

File: tests/partners/test_integration.py [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
 
 def deep_sort(d):
     """Sort all lists inside given dict, at all levels."""
     dict_copy = copy.deepcopy(d)
 
     for key, value in d.items():
         if isinstance(value, list):
-            dict_copy[key] = sorted(value)
+            dict_copy[key] = sorted(value, key=lambda item: (
+                tuple(item.items()) if isinstance(item, dict) else item))
```
Comment (Toshakins) [inline tests/partners/test_integration.py]: But on line 14 it explicitly checks that value if a list's instance. Does the dict inherit from the list or vice versa?

Comment (Toshakins) [reply tests/partners/test_integration.py]: ah, item is not the value

Comment (Toshakins) [reply tests/partners/test_integration.py]: Did that line work prior your change?

File: tests/accounts/base.py (hunk 2 above)
Comment (kosmos342) [reply tests/accounts/base.py]: when I switch to pytest it starts raise exception that this user already registered

File: tests/partners/test_integration.py (hunk 5 above)
Comment (kosmos342) [reply tests/partners/test_integration.py]: it works before with python2. in python3 it raises an exception because it can not compare dicts

File: tests/accounts/base.py (hunk 2 above)
Comment (Toshakins) [reply tests/accounts/base.py]: Wonder how it was working before 🤔

**Review by Toshakins — APPROVED** [review]
